src/Features/Core/Portable/Microsoft.CodeAnalysis.Features.csproj

```diff
@@ -16,6 +16,7 @@
   </PropertyGroup>
   <ItemGroup Label="Project References">
     <ProjectReference Include="..\..\..\Compilers\Core\Portable\Microsoft.CodeAnalysis.csproj" />
+    <ProjectReference Include="..\..\..\Compilers\CSharp\Portable\Microsoft.CodeAnalysis.CSharp.csproj" />
```

Comment:
This was not ok.  @maryamariyan this has to be removed as it is a layering violation.